web/hooks/installation.go

```diff
@@ -0,0 +1,61 @@
+package hooks
+
+import (
+	"context"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/qf"
+	"github.com/quickfeed/quickfeed/scm"
+)
+
+func (wh GitHubWebHook) handleInstallationCreated(event *github.InstallationEvent) {
+	installerID := uint64(event.GetSender().GetID())
+	user, err := wh.db.GetUserByRemoteIdentity(installerID)
```

Comment:
`s/user/courseCreator/`.

Should this user be checked if it is admin and allowed to create courses, ref. `interceptor/access_control.go`:
```json
	"CreateCourse":           {admin},
```
There is also this in `web/interceptor/tokens.go`:
```go
	"CreateCourse": // The signed in user gets the teacher role in the new course.
	func(ctx context.Context, tm *auth.TokenManager, _ userIDs) error {
		claims, ok := auth.ClaimsFromContext(ctx)
		if !ok {
			return fmt.Errorf("CreateCourse: missing claims in context")
		}
		return tm.Add(claims.UserID)
	},
```
Or can we trust that the event is from GitHub and representing the course creator?

Reply:
I believe we can trust that the event is coming from GitHub if we use [webhook secrets](https://docs.github.com/en/webhooks/using-webhooks/validating-webhook-deliveries).